app/Providers/BroadcastServiceProvider.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Pterodactyl\Providers;
+namespace App\Providers;
```

Comment:
Wrong namespace

Reply:
hadn't finished the namespace change yet.